src/elements/Button/Button.js

```diff
@@ -61,6 +62,8 @@ Button.propTypes = {
 	outline: PropTypes.bool,
 	/** Bare style */
 	bare: PropTypes.bool,
+	/** Text button styles */
+	text: PropTypes.bool,
```

Comment:
if we have a text-only button... is it still a button? What's the use case vs. an anchor with different styling?

Reply:
So it's for buttons like these:
<img width="296" alt="screen shot 2018-10-30 at 2 32 28 pm" src="https://user-images.githubusercontent.com/323984/47752088-c6774600-dc50-11e8-87f0-e23eddac67ca.png">

Which trigger actions, modals, etc. I was considering just using anchors at first, but these really do fit the bill as buttons more. But the variant style needed to be more of just a straight, "link style" button w/o all of the padding, background colors, etc. Maybe `text` isn't the best descriptor for it though

Reply:
Or, maybe this would make sore sense as an `IconButton`?

Reply:
I thought the `bare` button covered this? The icons I would definitely say just go inside with the text, so `IconButton` wouldn't make sense here.

```
<Button bare><Icon ...> Click me</Button>
```

Reply:
The `bare` style was kind of a different thing, but we don't actually use it anywhere / it's never appeared in any designs as it was. So yeah -- I've pushed an update just making the `bare` class do what we need